api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -37,7 +37,7 @@ function validatorReducer(network, validator) {
       validator.identity.web && validator.identity.web !== ``
         ? validator.identity.web
         : ``,
-    identity: identityReducer(validator.accountId, validator.identity),
+    identity: validator.identity.twitter,
```

Comment:
why twitter in particular? Is it the usually most complete one?

Reply:
We could use Twitter or Matrix. I don't know if Matrix actually has a profile picture.